packages/index.ts

```diff
@@ -53,6 +56,7 @@ export {
   SortableHeaderCell,
   NumberCell
 } from "./table";
+export { TabItem, TabTitle, Tabs } from "./tabs";
```

Comment:
Alphabetized! 😍

Reply:
Almost `Tabs` need to be before `TabTitle`.